### PR TITLE
Done homework 2

### PR DESCRIPTION
**Pull request by SamvelStepanyan92**

### CODE REVIEW

File: src/figureprinter/FigurePainter.java [hunk 1]
```diff
@@ -6,107 +6,173 @@
...
-     *       * *
-     *        *
-     * @param length  specified length of rhombus will be drawn
+     * @param length specified length of rhombus will be drawn
      */
     public static void drawRhombus(int length) {
-        // TODO : complete the method
+        for (int i = 0; i < length - 1; i++) {
+            for (int j = 0; j < (length - i - 1); j++) {
+                System.out.print("  ");
```
Comment (GevKishmishyan) [inline src/figureprinter/FigurePainter.java:116]: print one backspace - System.out.print(" ");

File: src/figureprinter/FigurePainter.java [hunk 2]
```diff
@@ -6,107 +6,173 @@
...
+            }
+            for (int j = 0; j < (i * 2 + 1); j++) {
```
Comment (GevKishmishyan) [inline src/figureprinter/FigurePainter.java:118]: **j < (i * 2 + 1)** - you don't need this, just put **j <= i**

File: src/figureprinter/FigurePainter.java [hunk 3]
```diff
@@ -6,107 +6,173 @@
...
+                System.out.print("* ");
+            }
+            System.out.println();
+        }
+        for (int i = 0; i < length; i++) {
+            for (int j = 0; j < i; j++) {
+                System.out.print("  ");
```
Comment (GevKishmishyan) [inline src/figureprinter/FigurePainter.java:125]: print one backspace - System.out.print(" ");

File: src/figureprinter/FigurePainter.java [hunk 4]
```diff
@@ -6,107 +6,173 @@
...
+            }
+            for (int j = 0; j < (length * 2 - i * 2 - 1); j++) {
```
Comment (GevKishmishyan) [inline src/figureprinter/FigurePainter.java:127]: **j < (length * 2 - i * 2 - 1)** - you don't need this, just put **j < length - i**

File: src/figureprinter/FigurePainter.java [hunk 5]
```diff
@@ -6,107 +6,173 @@
...
      * @param length specified length of triangle will be drawn
      */
-    public static void drawIsoscelesTriangle(int length){
-
+    public static void drawIsoscelesTriangle(int length) {
+        for (int i = 0; i < length; i++) {
+            for (int j = 0; j < (length - i - 1); j++) {
+                System.out.print("  ");
+            }
+            for (int j = 0; j < (i * 2 + 1); j++) {
```
Comment (GevKishmishyan) [inline src/figureprinter/FigurePainter.java:150]: the same is here

File: src/figureprinter/FigurePainter.java [hunk 6]
```diff
@@ -6,107 +6,173 @@
...
      * @param length specified length of triangle will be drawn
      */
-    public static void drawBottomIsoscelesTriangle(int length){
-
+    public static void drawBottomIsoscelesTriangle(int length) {
+        for (int i = 0; i < length; i++) {
+            for (int j = 0; j < i; j++) {
+                System.out.print("  ");
+            }
+            for (int j = 0; j < (length * 2 - i * 2 - 1); j++) {
```
Comment (GevKishmishyan) [inline src/figureprinter/FigurePainter.java:172]: the same is here

File: src/figureprinter/FigurePainter.java (hunk 2 above)
Comment (SamvelStepanyan92) [reply src/figureprinter/FigurePainter.java:118]: j<=i -ov logikan kpoxvi u patker@ sxal kstacvi

File: src/figureprinter/FigurePainter.java (hunk 1 above)
Comment (SamvelStepanyan92) [reply src/figureprinter/FigurePainter.java:116]: 2prabelne zut visual teski hamar e

File: src/figureprinter/FigurePainter.java (hunk 3 above)
Comment (SamvelStepanyan92) [reply src/figureprinter/FigurePainter.java:125]: the same is here

File: src/figureprinter/FigurePainter.java (hunk 4 above)
Comment (SamvelStepanyan92) [reply src/figureprinter/FigurePainter.java:127]: the same is here

File: src/figureprinter/FigurePainter.java (hunk 5 above)
Comment (SamvelStepanyan92) [reply src/figureprinter/FigurePainter.java:150]: the same is here

File: src/figureprinter/FigurePainter.java (hunk 6 above)
Comment (SamvelStepanyan92) [reply src/figureprinter/FigurePainter.java:172]: the same is here

File: src/figureprinter/FigurePainter.java (hunk 5 above)
Comment (SamvelStepanyan92) [reply src/figureprinter/FigurePainter.java:150]: estex 9hat el petk e vor tpe

**Pull request closed without merging**